### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ConfigurationProfile_Screenreader: Jaws doesn't announce, content of radiobutton

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9341]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopWeb;#Win32;#MAS1.3.1;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev2;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Additional Details:
MAS References: MAS1.3.1
JAWS Version: 2019.1912.1 ILM

**Additional Details:**
MAS reference: MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Launch JAWS screenereader
8. Navigate to any radiobutton and press down arrow key

**Actual:**
While JAWS is on and navigated to radiobutton and pressed down arrow key, screenreader doesn't announce content of radiobutton.

**Issue doesn't repro with NVDA screenreader.**

**Expected:**
While JAWS is on and navigated to radiobutton and pressed down arrow key, screenreader should announce content of radiobutton.

**User Impact:**
Screenreader users will face difficulty as they will not get the information present in radiobutton control.

**Attachment link reference:**
[9341_A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ConfigurationProfile_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274775/9341_A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ConfigurationProfile_Screenreader.zip)

**Comment by BhanuPrakashGit** [comment]
@alanrenmsft Could you please the reopened the bug as it should be verified the Trusted Tested.
Please provide the latest environment details where we can verify this and take action accordingly.

GithubTags:#Regressed:06-02-23;

**Comment by BhanuPrakashGit** [comment]
@alanrenmsft Tried to verify but not able to get "SQL Server Big Data Cluster" card under deployment options. Could you please look into this.

**Env Details:**
Version: 1.42.0-insider (system setup)
Commit: ac0dc4ee7627ba907c401c0c3d78746546b269b2
Date: 2023-02-06T06:50:44.371Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

![image](https://user-images.githubusercontent.com/104351566/217179916-7b28362f-0ac1-4e92-bab6-e5bc445dc036.png)

**Comment by alanrenmsft** [comment]
@BhanuPrakashGit as I mentioned in my previous emails, this feature has been removed, you can skip the issues for big data cluster feature.

**Comment by BhanuPrakashGit** [comment]
@alanrenmsft Thanks for the update.
**GithubTags**:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
